core/templates/pages/exploration-player-page/services/fatigue-detection.service.ts

```diff
@@ -43,8 +45,9 @@ export class FatigueDetectionService {
       this.windowStartTime = this.submissionTimesMsec.shift();
```

Comment:
What happens when undefined is returned?

Reply:
Returning false in that case